default-config/config

```diff
@@ -408,19 +408,17 @@ AddToMenu   MenuSendToPage
 # Fvwm Man Pages (Help) Menu
 DestroyMenu MenuFvwmManPages
 AddToMenu   MenuFvwmManPages "Help" Title
-+ "fvwm"                ViewManPage fvwm
++ "fvwm3"               ViewManPage fvwm3
 + "FvwmAnimate"         ViewManPage FvwmAnimate
 + "FvwmAuto"            ViewManPage FvwmAuto
 + "FvwmBacker"          ViewManPage FvwmBacker
 + "FvwmButtons"         ViewManPage FvwmButtons
-+ "FvwmCommand"         ViewManPage FvwmCommand
```

Comment:
This line should stay, since #393 will add a man page for this.

Reply:
Thanks, I just noticed the manpage was gone, but didn't realize a new one would replace it.

Updated.